docs/src/data/ignoredLinks.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * IGNORED_LINKS is a manually maintained allowlist.
+ * Any links on this list will not be tested, so KEEP THIS LIST AS SHORT AS POSSIBLE.
+ * Links added to this list because of one of the following reasons
+ * 1) they return unexpected status code
```

Comment:
Can you clarify what you mean by this? What's an unexpected status code?

Reply:
They may return 403 if they have a WAF blocking request from linux terminal, etc. I can clarify that in the comment

Reply:
Sorry, I meant could you clarify it in the comment?

Reply:
Done 861d272e5629dd335ce2e2fd48cd4ee6546243f8